Makefile

```diff
@@ -400,8 +404,11 @@ test_load_relays_stress_example: ## Run the stress test for E2E relays on a pers
 	-tags=load,test -run LoadRelays --log-level=debug --timeout=30m \
 	--manifest ./load-testing/loadtest_manifest_example.yaml
 
+.PHONY: test_load_relays_stress_stake_for_localnet
+test_load_relays_stress_stake_for_localnet:  gateway2_stake gateway3_stake supplier2_stake supplier3_stake ## Stakes all the actors necessary for a localnet relay stress test
+
```

Comment:
Can you elaborate on the rationale here? My understanding was that the load test does this in [`MoreActorsAreStakedAsFollows()`](https://github.com/pokt-network/poktroll/blob/main/load-testing/tests/relays_stress_test.go#L482) as it scales actors.